src/pages/Frontpage.jsx

```diff
@@ -13,6 +13,7 @@ function Frontpage(props) {
   const defaultValue2 = new Date(today).toISOString().split("T")[0];
   const datetimestr = `${defaultValue}T00:00`;
   const navigate = useNavigate();
+  const [owner, setOwner] = useState("");
```

Comment:
to podejście które tutaj zrobiłeś z osobnym stanem jest spoko, 

natomiast wykorzystana jest tu biblioteka do formularzy. Spróbuj zamiast tego stanu zarejestrować input z polem owner jak pola niżej